docs/integration/driver/windows.rst

```diff
@@ -0,0 +1,105 @@
+=============================
+Windows Display/Inputs driver
+=============================
+
+Overview
+-------------
+
+The **Windows** display/input `driver <https://github.com/lvgl/lvgl/src/dev/windows>`__ offers support for simulating the LVGL display and keyboard/mouse inputs in a Windows Win32 window.
+
+The main purpose for this driver is for testing/debugging the LVGL application in a **Windows** simulation window via **simulator mode**, or developing a standard **Windows** desktop application with LVGL via **application mode**.
+
+Here are the **similarity** for simulator mode and application mode.
+
+- Support LVGL pointer, keypad and encoder devices integration.
+- Support Windows touch input.
+- Support Windows input method integration input.
+- Support Per-monitor DPI Aware (both V1 and V2).
+
+Here are the **differences** for simulator mode and application mode.
+
+Simulator Mode
+^^^^^^^^^^^^^^
+
+- Designed for LVGL simulation scenario.
+- Keep the LVGL display resolution all time for trying best to simulate UI layout which will see in their production devices.
+- When Windows DPI scaling setting is changed, Windows backend will stretch the display content.
+
+Application Mode
+^^^^^^^^^^^^^^^^
+
+- Designed for Windows desktop application development scenario.
+- Have the Window resizing support and LVGL display resolution will be changed.
+- When Windows DPI scaling setting is changed, the LVGL display DPI value will also be changed.
+
+Prerequisites
+-------------
+
+The minimum Windows OS version this driver supported is Windows Server 2003, or Windows XP with East Asian language support installed because the input method integration support need that.
+
+Configure Windows driver
+--------------------
+
+Enable the Windows driver support in lv_conf.h, by cmake compiler define or by KConfig
+
+.. code:: c
+
+    #define LV_USE_WINDOWS  1
+
+Usage
+-----
+
+.. code:: c
+
+    #include <Windows.h>
+    #include "lvgl/lvgl.h"
+    #include "lvgl/examples/lv_examples.h"
+    #include "lvgl/demos/lv_demos.h"
+
+    int main()
+    {
+        lv_init();
+
+        int32_t zoom_level = 100;
+        bool allow_dpi_override = false;
+        bool simulator_mode = false;
+        lv_display_t* display = lv_windows_create_display(
+            L"LVGL Display Window",
+            800,
+            480,
+            zoom_level,
+            allow_dpi_override,
+            simulator_mode);
+        if (!display)
+        {
+            return -1;
+        }
+
+        lv_indev_t* pointer_device = lv_windows_acquire_pointer_indev(display);
```

Comment:
We don't use the `acquire` keyword now. What do you think about 

```suggestion
        lv_indev_t* pointer_device = lv_windows_get_pointer_indev(display);
```

Reply:
I think it should have some difference from the “get” used in LVGL APIs.

These APIs **get** the current indev related to specific display **if*** users already use these API before, or **create** a new instance and register.

Kenji Mouri

Reply:
I think the distinction needs to be emphasized. Because users can use lv_indev_delete to remove they acquired before and they can call these API to acquire the resources again if they want.

Reply:
@kisvegabor

Reply:
I understand now, let's keep using `acquire`.

Reply:
For my experience of using LVGL APIs, I found the get operations are get the current info directly and these operations are not consider about remove some info at runtime.

Kenji Mouri